sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -67,6 +67,30 @@ export function useFormioDropdownLoader() {
     );
   };
 
+  // Get Program description for the selected program and set to the field (programDesc) in formio
+  const loadProgramDesc = async (
```

Comment:
This class is for methods related to loading dropdowns. Loading this information does not sound like something directly related to the dropdowns loading. Yes, the data is loaded in reaction to a dropdown event, but still not loading a data to a dropdown.